src/orquestra/sdk/_ray/_build_workflow.py

```diff
@@ -343,7 +361,52 @@ def _import_pip_env(ir_invocation: ir.TaskInvocation, wf: ir.WorkflowDef):
             *(task_def.dependency_import_ids or []),
         )
     ]
-    return [chunk for imp in imports for chunk in _pip_string(imp)]
+
+    current_sdk_version: t.Optional[str] = _normalise_prerelease_version(
+        get_installed_version("orquestra-sdk")
+    )
```

Comment:
I'm not sure about this.
I think we should just ignore the dependency and not add it to the `pip_list` if it looks like a development version.
If we install a previous version, it's possible that you don't end up testing the feature you just developed.

Reply:
Good idea, [5631ffa](https://github.com/zapatacomputing/orquestra-workflow-sdk/pull/324/commits/5631ffab578770be00bd9d31838fa81b72a3ad8b) makes it so that dev versions of the sdk don't add a dependency to the tasks.